### PR TITLE
Do not rename main.css to main.min.css when minified

### PR DESCRIPTION
**Pull request by lmorchard**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -111,9 +111,8 @@ gulp.task('styles', ['sass-lint'], function stylesTask() {
     }).on('error', sass.logError))
     .pipe(autoprefixer('last 2 versions'))
       // don't minify in development
-      .pipe(gulpif(!IS_DEBUG, rename({ suffix: '.min' })))
```
Comment (pdehaan) [inline gulpfile.js:114]: Travis-CI says:

```
[21:54:54] ESLintError in plugin 'gulp-eslint'
Message:
    "rename" is defined but never used
Details:
    fileName: gulpfile.js
    lineNumber: 16
```

**Comment by lmorchard** [comment]
Updated to fix the lint failure

**Comment by pdehaan** [comment]
@lmorchard Not sure if you want to remove the **gulp-rename** devDep too.

**Comment by lmorchard** [comment]
Updated again

**Comment by jaredhirsch** [comment]
works and looks good :beers:

**Pull request merged**